docs/concepts/policy/pod-security-policy.md

```diff
@@ -1,97 +1,371 @@
-| The use of host's IPC namespace                                        | `hostIPC`                                   |
-| The SELinux context of the container                                   | [`seLinux`](#selinux)                       |
-| The user ID                                                            | [`runAsUser`](#runasuser)                   |
-| Configuring allowable supplemental groups                              | [`supplementalGroups`](#supplementalgroups) |
-| Allocating an FSGroup that owns the pod's volumes                      | [`fsGroup`](#fsgroup)                       |
-| Requiring the use of a read only root file system                      | `readOnlyRootFilesystem`                    |
-| Running of a container that allow privilege escalation from its parent | [`allowPrivilegeEscalation`](#allowprivilegeescalation) |
-| Control whether a process can gain more privileges than its parent process | [`defaultAllowPrivilegeEscalation`](#defaultallowprivilegeescalation) |
-| Whitelist of allowed host paths                                        | [`allowedHostPaths`](#allowedhostpaths)     |
-
-_Pod Security Policies_ are comprised of settings and strategies that
-control the security features a pod has access to. These settings fall
-into three categories:
-
-- *Controlled by a Boolean*: Fields of this type default to the most
-restrictive value.
-- *Controlled by an allowable set*: Fields of this type are checked
-against the set to ensure their values are allowed.
-- *Controlled by a strategy*: Items that have a strategy to provide
-a mechanism to generate the value and a mechanism to ensure that a
-specified value falls into the set of allowable values.
-
-
-## Strategies
-
-### RunAsUser
+| Control Aspect                                      | Field Names                                 |
+| ----------------------------------------------------| ------------------------------------------- |
+| Running of privileged containers                    | `privileged`                                |
+| Usage of the root namespaces                        | [`hostPID`, `hostIPC`](#host-namespaces)    |
+| Usage of host networking and ports                  | [`hostNetwork`, `hostPorts`](#host-namespaces) |
+| Usage of volume types                               | [`volumes`](#volumes-and-file-systems)      |
+| Usage of the host filesystem                        | [`allowedHostPaths`](#volumes-and-file-systems) |
+| Allocating an FSGroup that owns the pod's volumes   | [`fsGroup`](#volumes-and-file-systems)      |
+| Requiring the use of a read only root file system   | [`readOnlyRootFilesystem`](#volumes-and-file-systems) |
+| The user and group IDs of the container             | [`runAsUser`, `supplementalGroups`](#users-and-groups) |
+| Restricting root privileges                         | [`allowPrivilegeEscalation`, `defaultAllowPrivilegeEscalation`](#privilege-escalation) |
```

Comment:
Perhaps, it's better to rephrase "Restricting root privileges" to emphasize the fact that it not about root but about getting root privs.

What's about "Restricting obtaining of root privileges", "Restricting getting the root privileges"?

Reply:
Done.